internal/cloudflare.go

```diff
@@ -21,18 +21,23 @@ func SetupCloudflareClient() {
 	}
 }
 
-// GetCloudflareZoneID creates a global var containing the zone id
-func GetCloudflareZoneID() {
-	var err error
-	zoneID, err = cf.ZoneIDByName(config.Cloudflare.ZoneName)
-	if err != nil {
-		log.Fatalf("Unable to get Cloudflare zone id: %s", err)
+// GetCloudflareZoneIDs creates a global map containing the zone ids
```

Comment:
```suggestion
// GetCloudflareZoneIDs creates a global map containing the zone names and their zone ids
```
Maybe the functionname should be renamed, because the function now does more than only fetching the zone id's.
Purpose: CreateCloudflareZoneMap

Reply:
I will name it GetCloudflareZones